app/connector/support/maven-plugin/src/main/java/io/syndesis/connector/support/maven/GenerateConnectorInspectionsMojo.java

```diff
@@ -102,6 +117,78 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         }
     }
 
+    private void initJsonSchema() throws MojoExecutionException {
+        if (jsonSchema != null) {
+            return; // already initialised
+        }
+
+        InputStream schemaStream = null;
+        try {
```

Comment:
Use try-with-resources.

```suggestion
        try (InputStream schemaStream = GenerateConnectorInspectionsMojo.class.getResourceAsStream("/" + CONNECTOR_SCHEMA_FILE) {
```

Reply:
I did try that in both cases of accessing InputStreams. However, the code validations post-compile kept flagging them as possibilities for resource-leakage and failed the build. I looked into trying to ignore them / fix them but couldn't find anything obvious so reverted to a format that was guaranteed to work.